_includes/footer.html

```diff
@@ -54,23 +59,6 @@
   </ul>
 </footer>
 
-<script id="template-activity-item" type="text/template">
-  <li class="item">
-    <a href="https://github.com/<%= actor.login %>" class="avatar">
-      <img src="<%= actor.avatar_url %>">
-    </a>
-    <div class="content">
-      <a href="<%= link %>" class="name"> @
-        <%= actor.login %>
-      </a>
-      <%= event_type %>
-    </div>
-    <span class="date">
-      <%= created_at %>
-    </span>
-  </li>
-</script>
-
```

Comment:
Guess this was some outdated part where we could have shown the editor info? Fine to remove. :)